posthog/migrations/0711_alertconfiguration_notification_destinations.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.18 on 2025-04-21 12:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0710_add_salesforce_tables"),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
Typically we've tried to avoid this so far... We load related hog functions simply by using the hog function api to filter for things matching your filters. 

I'm not _super_ strongly tied to that opinion but doing it with a direct link would certainly be going slightly against the grain (we don't do this anywhere else like for actions or surveys)

Reply:
Thanks for pointing out, I've moved away from this db change/what error tracking are doing and the code is a lot simpler now. Works similar to surveys/actions now. Can see the slack notification coming in

<img width="722" alt="image" src="https://github.com/user-attachments/assets/c29c9e5f-5eb4-42c2-b662-a61485180883" />